### ISSUE TITLE
hydro

### ISSUE THREAD

**Issue by atenpas** [ros-perception/camera_pose#2]
will there be a version of this for hydro at some point?

**Comment by jbohren** [comment]
Yeah, #3 works with hydro. There are better solutions out there, though.

**Comment by Tones29** [comment]
jbohren, what other solutions do you recommend?
